main/test/ca/uwaterloo/flix/util/FlixSuite.scala

```diff
@@ -31,6 +31,15 @@ class FlixSuite(incremental: Boolean) extends AnyFunSuite {
     */
   var flix = new Flix()
 
+  def mkTestDirCollected(path: String)(implicit options: Options): Unit = {
+    val iter = Files.walk(Paths.get(path), 1)
```

Comment:
We have a class called something like FileOps or FileUtils. Could you move the Files.walk thingy to there?

I suppose it takes a path and returns a Scala list of paths.

Reply:
Sure thing :)